Src/DependencyCollector/Shared.Tests/Implementation/ProfilerHttpProcessingTest.cs

```diff
@@ -115,6 +115,27 @@ public void RddTestHttpProcessingProfilerOnExceptionForGetResponse()
         }
 
         /// <summary>
+        /// Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.
+        /// </summary>
+        [TestMethod]
+        [Description("Validates HttpProcessingProfiler sends correct telemetry including response code on calling OnExceptionForGetResponse for WebException.")]
+        [Owner("cithomas")]
```

Comment:
Just FYI, we do not use owners attribute. These are old tests and attributes were added by the previous owner team. We can safely remove it for all the old tests and do not add for the new ones. (I'm not asking to do that now, just commenting for the future.)

---
In reply to: [66872558](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/94#discussion_r66872558) [](ancestors = 66872558)